docs/test_instructions_per_dependency_update.md

```diff
@@ -0,0 +1,859 @@
+- (STOP)
+
+Step.2:
+- Head over to your desktop and open a web browser (note that using an incognito tab works best).
+- Browse to `wordpress.com` (note that if you are logged-in, log-out first).
+- Tap the `Log In` link (top-right).
+- Tap the `Login via the mobile app` link in the list of options below the main `Continue` button
+  (bottom-middle).
+- Verify you are on the `Login via the mobile app` view and `Use QR Code to login` is shown, along
+  with a QR code for you to scan.
+- (STOP)
+
+Step.3:
+- Head back to your mobile.
+- Tap the `Scan Login Code` item on the `Me` screen you are currently at.
+- Scan the QR code on the web browser.
+- Follow the remaining prompts on your mobile to login to WordPress on your web browser (desktop),
+  verify that you have successfully logged-in and are able to use WordPress as expected.
+
+</details>
+
+-----
+
+### Okio [[squareupOkioVersion](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/build.gradle#L1)] <a name="okio"></a>
+
+<details>
+    <summary>1. [JP/WP] Me Screen [GravatarApi.java + StreamingRequest.java]</summary>
+
+- Go to `Me` tab.
+- From the `Me` screen you are in, click on your profile's icon (`CHANGE PHOTO`).
+- Choose an image and wait for the `Edit Photo` screen to appear.
+- Crop the image and click the `done` menu option (top right).
+- Verify the image is updated accordingly.
+
+</details>
+
+-----
+
+### Zendesk [[zendeskVersion](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/build.gradle#L1)] <a name="zendesk"></a>
+
+<details>
+    <summary>1. [JP] Zendesk Update [Tickets & Contact Support]</summary>
+
+ℹ️ This test only works when testing with a normal, non-a8c user account.
+
+- Go to `Me` tab -> `Help` -> `Tickets`.
+- Verify that the `My tickets` Zendesk related screen opens and that it lists all of your tickets
+  (or non if you don't have any yet).
+- Go back to the `Help` settings screen and then click on `Contact Support`.
+- Verify that the `Help` Zendesk related screen opens and that you are prompt to `Send a message...`.
+- Type `Testing Zendesk X.Y.Z update, please ignore and thank you!` and then send your message.
+- Go back to the `Help` settings screen and then click on `Tickets` again.
+- Verify that your previously sent message is listed in there and that you can click on it.
+  PS: You could also check your emails and verify that you got a
+  `Thank you for contacting the WordPress.com support team! (#1234567)` email.
+- Verify that clicking on it navigates you to the inner screen where your message is shown as
+  `Delivered` along with a predefined automated `mobile support` response message.
+
+</details>
+
+-----
+
+### JSoup [[jsoupVersion](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/build.gradle#L1)] <a name="jsoup"></a>
+
+<details>
+    <summary>1. [JP/WP] Classic Editor [Aztec]</summary>
+
+ℹ️ Disable the `Block Editor` before testing this.
```

Comment:
Can we add steps for disabling and enabling `Block Editor`?

Reply:
Good idea! 💯

Reply:
Done: f3870ce23a6e1a3587dab105bb199da5cb7b2b3e